doc/source/conf.py

```diff
@@ -34,7 +34,7 @@
 # cross-reference to the Python function “filter”. The default is None, which doesn’t
 # reassign the default role.
 
-default_role = "py:obj"
+default_role = "code"
```

Comment:
perhaps add what you have in the PR description as a comment here

what is the syntax for adding reference now (if not single backtick)

Reply:
> perhaps add what you have in the PR description as a comment here

Will do!

> what is the syntax for adding reference now (if not single backtick)

See the sphinx documentation for referencing syntax: https://www.sphinx-doc.org/en/master/usage/referencing.html